sklearn/neighbors/nca.py

```diff
@@ -3,7 +3,9 @@
 Neighborhood Component Analysis
 """
 
-# License: BSD 3 Clause
+# Authors: William de Vazelhes <wdevazelhes@gmail.com>
+#          John Chiotellis <ioannis.chiotellis@in.tum.de>
```

Comment:
@johny-c I put the email I found on your webpage, but feel free to put another one here ;)